src/app/fyle/spender-onboarding/spender-onboarding.page.spec.ts

```
@@ -139,9 +141,59 @@
     });
   });
 
+  it('goBackToConnectCard(): should set currentStep to OnboardingStep.CONNECT_CARD when goBackToConnectCard is called', () => {
+    component.goBackToConnectCard();
+
+    expect(component.currentStep).toBe(OnboardingStep.CONNECT_CARD);
+  });
+
+  fdescribe('setPostOnboardingScreen(): ', () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)